classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialized(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student) << student
+  end
```

Comment:
- [ ] I think that using ```push``` and ```<<``` will do the same thing hence this relationship would not be set up correctly. Kindly ensure that the classroom is properly set for the student. You can read more on how this can be achieved [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)

Reply:
@Ghiftee  Many Thanks!